glommio/src/channels/shared_channel.rs

```diff
@@ -393,13 +393,25 @@ impl<T: Send + Sized> ConnectedReceiver<T> {
     }
 
     fn recv_one(&self, cx: &mut Context<'_>) -> Poll<Option<T>> {
```

Comment:
I think we can remove this and just recurse on `do_recv_one()`.

Reply:
The `disconnected` param of `do_recv_one` is only for internal use, not necessarily being exposed. And it should impose no runtime overhead because of inlining.